cmd/tink-cli/cmd/completion.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 // completionCmd returns the completion command that, when run, generates a
-// bash or zsh completion script for the CLI
-func completionCmd(name string) *cobra.Command {
+// bash or zsh completion script for the CLI.
+func completionCmd(_ string) *cobra.Command {
```

Comment:
the initial purpose of this was to plumb the command name into the completion help text. I recall being near this but I don't recall why it wasn't done.